File: myphotoviewer-util/src/main/java/dev/novokrest/myphotoviewer/util/filesystem/Verifiers.java
package dev.novokrest.myphotoviewer.util.filesystem;


public class Verifiers {

    public static void Verify(boolean b) {
        if (!b) {
            throw new RuntimeException();
        }
    }

    public static void Verify(boolean b, String message, Object... args) {
        if (!b) {
            throw new RuntimeException(String.format(message, args));
        }
    }
}

File: windows/MyPhotoViewer/MyPhotoViewer.Core/Verifiers.cs
using System;

namespace MyPhotoViewer.Core
{
    public static class Verifiers
    {
        public static void Verify(bool b, string messageFormat, params object[] objects)
        {
            if (!b)
            {
                throw new Exception(string.Format(messageFormat, objects));
            }
        }

        public static void Verify(bool b, Func<Exception> exceptionCreator)
        {
            if (!b)
            {
                throw exceptionCreator();
            }
        }

        public static void ArgVerify(bool b, string paramName, string messageFormat = "", params object[] objects)
        {
            if (!b)
            {
                throw new ArgumentException(string.Format(messageFormat, objects), paramName);
            }
        }

        public static void ArgNullVerify(object arg, string argName)
        {
            if (arg == null)
            {
                throw new ArgumentNullException(nameof(argName));
            }
        }
    }
}